storage/time-series/influxdb/retriever/app/express/index.js

```diff
@@ -73,17 +73,15 @@ module.exports = (routes, serviceState, openApiPath) => {
         stateManager: serviceState,
         logger,
       }),
-      responseCompressInterceptor({
-        supportTrustProxy: configExpress.trustproxy,
-      }),
+      responseCompressInterceptor({}),
```

Comment:
You don't need to pass anything as a parameter that by default an empty object is created